### PR TITLE
Cleanup System.Private.ServiceModel.csproj

### PR DESCRIPTION
**Pull request by roncain**
Added AssemblyVersion 4.0.0.0 to be consistent with CoreFx.
Removed CodeAnalysisRuleSet to eliminate a warning building mirrored sources in TFS.
Removed unnecessary source control properties.

### CODE REVIEW

**Comment by SajayAntony** [comment]
:shipit:

File: src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj [hunk 1]
```diff
@@ -5,25 +5,17 @@
     <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
     <Platform Condition="'$(Platform)' == ''">AnyCPU</Platform>
     <AssemblyName>System.Private.ServiceModel</AssemblyName>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <RootNamespace>System.ServiceModel</RootNamespace>
     <OutputType>Library</OutputType>
     <NoWarn>$(NoWarn);1634;1691;649</NoWarn>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
```
Comment (iamjasonp) [inline src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj]: Remind me: `CodeAnalysisRuleSet` means "FxCop", right?

**Comment by roncain** [comment]
Yes, it means FxCop.  We originally added it as a starting point but this now causes issues building from mirrored sources

**Comment by iamjasonp** [comment]
Thanks for the explanation, @roncain. 

ACK :shipit:

**Pull request merged**